### PR TITLE
Searchbar

### PR DESCRIPTION
**Pull request by eileenfuerstenau**

### CODE REVIEW

File: src/components/BookCardsPage/BookCardsPage.js [hunk 1]
```diff
@@ -76,12 +84,12 @@ const CardsPageLayout = styled.div`
...
-    padding-top: 55px;
+    padding-top: 105px;
   }
 `
-const ButtonWrapper = styled.div`
-  justify-content: space-around;
-  display: flex;
+const TabBarWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 50% 50%;
```
Comment (mndys) [inline src/components/BookCardsPage/BookCardsPage.js:92]: Just an alternative way of writing this, but maybe you'd like to use fractions here instead? If you ever wanted a third or fourth row (perhaps on bigger screens?) you wouldn't have to manually adjust percentages, if you wrote `grid-template-columns: 1fr 1fr`.

File: src/components/SearchBar/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+export default function SearchBar({ userInput, setUserInput }) {
+  return (
+    <InputLabel>
+      <Icon glyph="search" size={30} />
+      <Input
+        value={userInput}
+        onChange={event => setUserInput(event.target.value)}
+        maxLength={200}
+        placeholder="Search books"
```
Comment (mndys) [inline src/components/SearchBar/SearchBar.js:13]: I hope I haven't misread it, but I've seen in a filter on the BookCardsPage, that your search accepts both author's names and book titles. You could let your users know via the placeholder text, that they can search for either here.

**Review by mndys — APPROVED** [review]
I really like your app so far! The search function works well, runs fast (I'd say this counts as your QA review done as well 😉 ) and the code is nice and clean. :) Really looking forward to seing your app evolve! 🎉

File: src/components/SearchBar/SearchBar.js (hunk 2 above)
Comment (eileenfuerstenau) [reply src/components/SearchBar/SearchBar.js:13]: Great idea, ill do so. Thanks!

**Pull request merged**